docs/source/tutorials/lora_finetune.rst

```diff
@@ -315,13 +318,13 @@ to see our peak memory during a finetune. We will experiment along two axes:
 first, which model layers have LoRA applied, and second, the rank of each LoRA layer. (We will scale
 alpha in parallel to LoRA rank, as discussed above.)
 
-To compare the results of our experiments, we can evaluate our models on :code:`truthfulqa_mc2`, a task from
+To compare the results of our experiments, we can evaluate our models on `truthfulqa_mc2 <https://github.com/sylinrl/TruthfulQA>`_, a task from
 the `TruthfulQA <https://arxiv.org/abs/2109.07958>`_ benchmark for language models. For more details on how to run this and other evaluation tasks
 with torchtune's EleutherAI evaluation harness integration, see our :ref:`End-to-End Workflow Tutorial <eval_harness_label>`.
 
 Previously, we only enabled LoRA for the linear layers in each self-attention module, but in fact there are other linear
 layers we can apply LoRA to: MLP layers and our model's final output projection. Note that for Llama-2-7B the final output
-projection maps to dimension 32000 (instead of 4096 as in the other linear layers), so enabling LoRA for this layer will increase
+projection maps to the vocability dimension (32000 instead of 4096 as in the other linear layers), so enabling LoRA for this layer will increase
```

Comment:
just confirming (english is not 1st language), is vocabilty the right word? Should it be vocabulary?

Reply:
your English is perfect Felipe! it's a good catch